### PR TITLE
Fix bad hash in web/javascript/reference/errors/not_defined

### PR DESCRIPTION
**Pull request by queengooborg**
This PR fixes a bad hash in a link across all locales.

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/errors/not_defined/index.md [hunk 1]
```diff
@@ -67,4 +67,4 @@ console.log(num1); // 2
 
 - {{Glossary("Scope")}}
 - [Declaring variables in the JavaScript Guide](/zh-CN/docs/Web/JavaScript/Guide/Grammar_and_types#Declaring_variables)
-- [Function scope in the JavaScript Guide](/zh-CN/docs/Web/JavaScript/Guide/Functions#Function_scope/en-US/docs/)
+- [Function scope in the JavaScript Guide](/zh-CN/docs/Web/JavaScript/Guide/Functions#Function_scope)
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/errors/not_defined/index.md:70]: We have translated the heading in zh-CN :)

```suggestion
- [Function scope in the JavaScript Guide](/zh-CN/docs/Web/JavaScript/Guide/Functions#函数作用域)
```

File: files/ko/web/javascript/reference/errors/not_defined/index.md [hunk 2]
```diff
@@ -67,4 +67,4 @@ console.log(num1); // 2
 
 - {{Glossary("Scope")}}
 - [Declaring variables in the JavaScript Guide](/en-US/docs/Web/JavaScript/Guide/Grammar_and_types#Declaring_variables)
-- [Function scope in the JavaScript Guide](/en-US/docs/Web/JavaScript/Guide/Functions#Function_scope/en-US/docs/)
+- [Function scope in the JavaScript Guide](/en-US/docs/Web/JavaScript/Guide/Functions#Function_scope)
```
Comment (yin1999) [inline files/ko/web/javascript/reference/errors/not_defined/index.md:70]: I've used google translate to check the hash :)

```suggestion
- [Function scope in the JavaScript Guide](/ko/docs/Web/JavaScript/Guide/Functions#함수_스코프)
```

File: files/ja/web/javascript/reference/errors/not_defined/index.md [hunk 3]
```diff
@@ -69,4 +69,4 @@ console.log(numbers()); // 5
 
 - [スコープ](/ja/docs/Glossary/Scope)
 - [変数の宣言 (JavaScript ガイド)](/ja/docs/Web/JavaScript/Guide/Grammar_and_types#declaring_variables)
-- [関数のスコープ (JavaScript ガイド)](/ja/docs/Web/JavaScript/Guide/Functions#function_scope/en-us/docs/)
+- [関数のスコープ (JavaScript ガイド)](/ja/docs/Web/JavaScript/Guide/Functions#function_scope)
```
Comment (yin1999) [inline files/ja/web/javascript/reference/errors/not_defined/index.md:72]:
```suggestion
- [関数のスコープ (JavaScript ガイド)](/ja/docs/Web/JavaScript/Guide/Functions#関数のスコープ)
```

File: files/es/web/javascript/reference/errors/not_defined/index.md [hunk 4]
```diff
@@ -65,4 +65,4 @@ console.log(num1); // 2
 
 - {{Glossary("Ámbito")}}
 - [Guía; declarando variables en JavaScript](/es/docs/Web/JavaScript/Guide/Grammar_and_types#Declaring_variables)
-- [Guía; contexto de la función en Java Script](/es/docs/Web/JavaScript/Guide/Functions#Function_scope/en-US/docs/)
+- [Guía; contexto de la función en Java Script](/es/docs/Web/JavaScript/Guide/Functions#Function_scope)
```
Comment (yin1999) [inline files/es/web/javascript/reference/errors/not_defined/index.md:68]:
```suggestion
- [Guía; contexto de la función en Java Script](/es/docs/Web/JavaScript/Guide/Functions#ámbito_de_function)
```

**Review by yin1999 — COMMENTED** [review]
I've used google translate to check other local's hash (might be wrong, feel free to correct them :))

File: files/ko/web/javascript/reference/errors/not_defined/index.md (hunk 2 above)
Comment (queengooborg) [reply files/ko/web/javascript/reference/errors/not_defined/index.md:70]: Looks like that's the correct header!

**Review by yin1999 — APPROVED** [review]
LGTM for zh-CN. Thanks :)

**Review by mfuji09 — APPROVED** [review]
LGTM for 'ja'.
Thanks for @queengooborg and @yin1999 !

**Comment by queengooborg** [comment]
I'm going to go ahead and merge this, since I've triple-checked to make sure they're the right anchors for all locales and have gotten approval from a few locales already!

**Comment by Graywolf9** [comment]
Sorry about the late, and thank you so much @queengooborg and @yin1999

**Pull request merged**